MODULE.bazel

```diff
@@ -68,15 +68,20 @@ pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
 use_repo(pip, "pip_deps")
 
 crate = use_extension("@rules_rust//crate_universe:extension.bzl", "crate")
```

Comment:
isn't this code gone? https://github.com/protocolbuffers/protobuf/pull/18650/files

Reply:
oh I see it's removed in 29.x but not main...

Reply:
Sorry I'm not familiar with the branching / PR process here ! I guess this PR doesn't make sence if it's going to be removed in 29.x !

Reply:
I'm not sure, I've never worked in this repo, I was also confused that the commit landed to a release branch but not main :)

Reply:
Yeah, sorry that part is confusing: the code is temporarily removed for the 29 release towards debloating the release for people who don't use Rust. It was made directly on the 29 release branch and not made on main since there's ongoing work that will decide how to avoid some of the bloat concerns for our 30+ releases.

So its worth still looking at improvements here, though know that its not being included for 29 and things made also change for 30+